### PR TITLE
PUB-1362 ET Daily List PDF and artefact summary generation

### PR DESCRIPTION
**Pull request by KianKwa**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-1362

### Change description ###

ET Daily List PDF and artefact summary generation

image

image

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/artefactsummary/EtDailyList.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+        StringBuilder output = new StringBuilder(140);
+        jsonPayload.get("courtLists").forEach(courtList -> {
+            courtList.get("courtHouse").get("courtRoom").forEach(courtRoom -> {
+                courtRoom.get("session").forEach(session -> {
+                    session.get("sittings").forEach(sitting -> {
+                        sitting.get("hearing").forEach(hearing -> {
+                            hearing.get("case").forEach(hearingCase -> {
+                                output
+                                    .append("\t•Start Time: ")
```
Comment (ChrisS1512) [inline src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/artefactsummary/EtDailyList.java:26]: I think you can use GeneralHelper.appendToStringBuilder for the below (it also handles nulls)

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/artefactsummary/EtDailyList.java (hunk 1 above)
Comment (KianKwa) [reply src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/artefactsummary/EtDailyList.java]: Updated

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/filegeneration/helpers/listmanipulation/EtDailyListManipulation.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            }
+        });
+
+        ((ObjectNode) hearing).put(CLAIMANT, claimant.get());
+        ((ObjectNode) hearing).put(CLAIMANT_REPRESENTATIVE, claimantRepresentative.get());
+        ((ObjectNode) hearing).put(RESPONDENT, respondent.get());
+        ((ObjectNode) hearing).put(RESPONDENT_REPRESENTATIVE, respondentRepresentative.get());
+    }
+
+    private static String createIndividualDetails(JsonNode party) {
```
Comment (ChrisS1512) [inline src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/filegeneration/helpers/listmanipulation/EtDailyListManipulation.java:75]: Could we use the createIndividualDetails from Data Manipulation here? It does look a bit different but potentially does the same

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Review by joshblackmoor — APPROVED** [review]
LGTM 👍

**Pull request merged**